zipkin-storage/mysql-v1/src/main/resources/mysql.sql

```diff
@@ -42,7 +41,6 @@ CREATE TABLE IF NOT EXISTS zipkin_dependencies (
   `parent` VARCHAR(255) NOT NULL,
   `child` VARCHAR(255) NOT NULL,
   `call_count` BIGINT,
-  `error_count` BIGINT
+  `error_count` BIGINT,
+  PRIMARY KEY (`day`, `parent`, `child`)
 ) ENGINE=InnoDB ROW_FORMAT=COMPRESSED CHARACTER SET=utf8 COLLATE utf8_general_ci;
-
-ALTER TABLE zipkin_dependencies ADD UNIQUE KEY(`day`, `parent`, `child`);
```

Comment:
this syntax was crashing initialization on the docker container.. not sure why, but anyway fixed now